Dockerfile

```diff
@@ -1,20 +1,28 @@
 # syntax=docker/dockerfile:1
 
-FROM python:3.7-slim-buster as builder
+ARG  PY_VERSION=3.10
+FROM python:${PY_VERSION}-buster as builder
 
-WORKDIR /app
+WORKDIR /py4ai-core
+
+RUN apt-get update && apt-get upgrade -y
 
 COPY requirements/requirements.in requirements/requirements.in
-COPY requirements/requirements_dev.txt requirements/requirements_dev.txt
-COPY LICENSE MANIFEST.in versioneer.py setup.py pyproject.toml README.md ./
+COPY requirements/requirements_dev.in requirements/requirements_dev.in
```

Comment:
This should be the txt, in order to have a reproducible and very stable env. Also the txt should be in the MANIFEST right?

Reply:
Done